testsuite/integration/basic/src/test/java/org/wildfly/test/integration/microprofile/health/MicroProfileHealthTestBase.java

```diff
@@ -60,15 +59,37 @@
 @RunAsClient
 public abstract class MicroProfileHealthTestBase {
```

Comment:
This test should not be modified, it verifies the behaviour for a WAR deployment.
Checking EAP  deployment should be done by another test